src/ad-bidders/prebid/intent-iq/intent-iq.ts

```diff
@@ -55,16 +60,27 @@ export class IntentIQ {
 				? 'fandom.com'
 				: window.location.hostname;
 
+			const intentIqConfig = {
+				partner: this.fandomId,
+				pbjs,
+				timeoutInMillis: DEFAULT_MAX_DELAY,
+				ABTestingConfigurationSource: 'percentage',
+				abPercentage: 97,
+				manualWinReportEnabled: true,
+				domainName,
+			};
+
+			if (this.intentIQVersion === newScriptVersion) {
+				intentIqConfig['vrBrowserBlackList'] = ['chrome'];
+			} else {
+				intentIqConfig['browserBlackList'] = 'Chrome';
```

Comment:
(nit) 
```suggestion
				intentIqConfig.browserBlackList = 'Chrome';
```
I'm wondering why linter did not pick this up 🤔